src/components/rating/rating.tsx

```diff
@@ -176,7 +216,7 @@ export class Rating
                 <calcite-icon icon="star-f" scale={this.scale} />
               </div>
             )}
-            <span class="visually-hidden">{this.intlStars.replace("${num}", `${i}`)}</span>
+            <span class="visually-hidden">{this.messages.stars.replace("${num}", `${i}`)}</span>
```

Comment:
for variables in messages, do they prefer `{}` or `${}`? Should we make one or the other a preferred best practice?

Reply:
It's more of a preference. Using `${ }` makes it easier to understand that it is a custom value. I am open to use `{ }` as well.

Reply:
cool is that in the conventions file?

Reply:
nope, this is not in the conventions yet.